examples/org.eclipse.glsp.example.workflow/src/org/eclipse/glsp/example/workflow/layout/WorkflowLayoutEngine.java

```diff
@@ -27,12 +29,16 @@
    @Inject
    protected GModelState modelState;
 
+   @Inject
+   protected ActionDispatcher actionDispatcher;
+
    @Override
    public void layout() {
       if (modelState.getRoot() instanceof GGraph) {
          GLSPLayoutConfigurator configurator = new GLSPLayoutConfigurator();
          configurator.configureByType("graph");
          this.layout((GGraph) modelState.getRoot(), configurator);
+         actionDispatcher.dispatchAfterNextUpdate(new CenterAction());
```

Comment:
I wouldn't expect that the layout engine queues an action after layouting -- so maybe this isn't the best example use case for `dispatchAfterNextUpdate`.

Reply:
I personally think it's not an uncommon usecase to want  to trigger a Center action after (manually) autolayouting the model. However, I happily change this to a better example. Any suggestions?

Reply:
An Idea could be: Select a node after the creation operation.

Reply:
The use case to center after layout makes sense, but I find it a bit odd that the layout engine invokes this action. I'd rather expect the center action to be invoked from the layout handler. But I'm fine keeping it this way.
I like the idea by @ivy-cst though.

Reply:
I also like the idea by @ivy-cst and updated the PR accordingly.